### DISCUSSION TITLE
BepInEx will not produce plugins folder.

### DISCUSSION THREAD

**Discussion by AkhiMertail** [BepInEx/BepInEx#257] [Help and Q&A]
I've tried downloading both the 5.4.9 and 5.4.8 releases directly from their github page, extracted all the contents of the .zip file into the root directory. I started my game and exited out but both tries now and the plugins folder and all the extra config files that BepInEx should generate are not there. Is anybody else having this issue? I'm running Windows 10 64 bit.

**Comment by fromda** [comment]
I'm having this issue. It seems that nobody has yet to resolve this issue, or are not trying to. I've done everything i can and i still haven't found the reason for this issue. If anyone comes across this and knows what to do, please comment the solution as i've been struggling for the past week over this.

**Comment by Liven28** [comment]
Did you try both x64 and x86 version of BepInEx
The one working doesn't depend on you windows but on the game itself (some games run in x86 in win x64)
